migrate/labels.yml

```diff
@@ -0,0 +1,118 @@
+- name: "help wanted"
+  color: "54976d"  # fenway green
+  description: "Ready to be picked up by anyone in the community"
+
+
+### LABELS INDICATING BROAD THEMES OF WORK.
+### MORE THAN ONE OF THESE MAY APPLY AT A TIME.
+
+- name: "maintenance"
+  color: "ff9125"  # construction orange
+  description: "Relates to platform health and/or routine upkeep"
+
+- name: "security"
+  color: "f0d165"  # caution yellow
+  description: "Relates to improving to the security posture of the platform"
+
+- name: "enhancement"
+  color: "a2eeef"  # electric blue!
+  description: "Relates to new features or improvements to existing features"
+
+- name: "documentation"
+  color: "0052cc"  # royal blue
+  description: "Relates to documentation improvements"
+
+
+### LABELS INDICATING THE SCOPE OR FUNCTION OF THE ISSUE.
+### AT MOST ONE OF THESE SHOULD BE USED AT A TIME.
+
+- name: "depr"
+  color: "3c3cc3"  # indigo-blue
+  description: "Proposal for deprecation & removal per OEP-21"
+
+- name: "bug"
+  color: "d93f0b"  # scarlet red...
+  description: "Report of or fix for something that isn't working as intended"
+
+- name: "discovery"
+  color: "d876e3"  # a curious shade of lavendar
+  description: "Pre-work to determine if an idea is feasible"
+
+- name: "epic"
+  color: "7e027e"  # epic violet
+  description: "Large unit of work, consisting of multiple tasks"
+
+- name: "initiative"
+  color: "340963"  # formidably deep mauve
+  description: "Huge unit of work, consisting of multiple epics"
+
+
+### LABELS INDICATING ISSUE OUTCOMES.
+
+- name: "wontfix"
+  color: "ffffff"  # white
+  description: "This will not be worked on"
+
+- name: "duplicate"
+  color: "cfd3d7"  # duplicitous grey
+  description: "This issue or pull request already exists elsewhere"
+
+
+### LABELS USED IN THE OPEN EDX CONTRIBUTION WORKFLOW.
+### PLEASE CONSULT WITH OPEN EDX PROJECT MANAGMENT BEFORE CHANGING THESE.
+
+- name: "waiting on author"
+  color: "bfd6f6"  # baby blue
+  description: "PR reviewers are waiting on the author to answer questions, fix tests, etc."
+
+- name: "inactive"
+  color: "cc950a"  # clay-ish brown
+  description: "PR author has been unresponsive for several months"
+
+- name: "closed inactivity"
+  color: "dbcd00"  # gold
+  description: "PR was closed because the author abandoned it"
+
+- name: "needs test run"
+  color: "f5424b"  # crimson red
+  description: "Author's first PR to a repository, awaiting test authorization from Axim"
+
+- name: "open source contribution"
+  color: "f0f0f0"  # light grey
+  description: "PR author is not from Axim or 2U"
+
+- name: "blocked by other work"
+  color: "ffa500"  # yellow orange
+  description: "PR cannot be finished until other work is complete"
+
+- name: "product review"
+  color: "c97bf7"  # light puple
+  description: "PR requires product review before merging"
+
+- name: "core contributor"
+  color: "000000"  # black
+  description: "PR is Core Contributor with rights in this repository (manually added)"
```

Comment:
another wording option: "PR **author** is a CC..."

Reply:
@itsjeyd I haven't used the label at all myself. What I have listed in the description matches the intended behavior of the automatically-added `core committer` label, although that stopped working reliably a year or two ago.

As far as I can tell, the label exists to smooth the contribution workflow for CCs. What do you and @mphilbrick211 think is the best behavior? Whatever it is, I'm happy to describe that here. If the label seems unnecessary, that is OK too; we can remove it.

Reply:
@itsjeyd @kdmccormick for the purposes of the OSPRs on the Contributions board, I have been using it if the author of the PR is a CC (and I've seen CCs who add that label for that reason as well). Typically if I see that a CC is the author of a PR, it lets me know that that PR will generally be handled in a timely manner, and I may not check in on those as frequently as others. This is because other CCs tend to jump in and help provide feedback and review on those, and the process tends to move fairly quickly.

I agree that it would be good to solidify what that label is used for so it's not confusing. 

On a related note, is there a way to tell which CCs have rights in each repo?

Reply:
@mphilbrick211 

> On a related note, is there a way to tell which CCs have rights in each repo?

I'm not sure if we have a single source of truth that's always up-to-date for each CC. (I've mainly been referencing the "Project(s) Working On" column from [this wiki page](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3156344833/Core+Contributors+to+the+Open+edX+Project) when trying to check but the info listed there isn't exhaustive for each CC and doesn't follow a consistent format.)

If I understand correctly, that was one of the reasons for retiring the workflow that would add the old `core committer` label to PRs automatically.

CC @kdmccormick

Reply:
@itsjeyd @mphilbrick211 Sounds good! I'll update the description to indicate that `core contributor` means that the author is generally a CC, even if they don't have rights to the specific repo against which the PR is open.

> I'm not sure if we have a single source of truth that's always up-to-date for each CC. (I've mainly been referencing the "Project(s) Working On" column from [this wiki page](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3156344833/Core+Contributors+to+the+Open+edX+Project) when trying to check but the info listed there isn't exhaustive for each CC and doesn't follow a consistent format.)

@itsjeyd The wiki page is supposed to be the source-of-truth; it's what Axim refers to when we are updating or auditing a CC's access rights in GitHub. If it's out of date then that's something we should fix! Do you see any examples on that page that need fixing?

Reply:
@kdmccormick 

> The wiki page is supposed to be the source-of-truth; it's what Axim refers to when we are updating or auditing a CC's access rights in GitHub.

Ah OK 💡 Good to know. 

> If it's out of date then that's something we should fix! Do you see any examples on that page that need fixing?

I can't point out any specific examples in terms of things that are out of date (seeing as I don't know each and every CC's repo access levels 😅).

A more general question would be: Is there a defined process for making sure that the contents of the page are up-to-date?

In terms of concrete things to fix:

* The list of repos is not always exhaustive. Example (Jenna):
    > Modular Learning, Product Core, **many others**

* ... Or it's not clear where to find a detailed list of repos if needed. Example (Viktor, Matjaz):
    > Paragon Blended Support and Maintenance

    > All repos currently in the latest Named Release

* Sometimes it's not clear what types of access a CC has. Example (Michelle):
    > All repos pull requests!

Reply:
Agreed with @itsjeyd - @kdmccormick sometimes that wiki page only lists areas of focus/expertise, and not the repo(s) the CC has rights in. It would be great if it DID list that!

Reply:
Yes, updating it is part of our CC onboarding & offboarding runbooks: https://openedx.atlassian.net/wiki/spaces/COMM/pages/3359342743/Participant+Administrator+Runbooks

Thanks for that feedback. Non-coding CCs are not granted write access to any repositories, so their "Projects Worked On" column is more informational than authoritative; we should make that clearer. As for coding CCs with rights like "All repos currently in the latest Named Release", that does mean something specific to Axim engineers, but I can see how it's opaque to others.

I wonder if we can avoid any ambiguity by linking you directly to permission listings on GitHub. @itsjeyd , can you see either of these pages, or do you get a 404?
* https://github.com/orgs/openedx/teams/frontend-all
* https://github.com/orgs/openedx/people/brian-smith-tcril

Reply:
@kdmccormick FWIW, I can see the frontend-all one, but not Brian's

Reply:
Good to know @mphilbrick211 !

Reply:
@mphilbrick211 @itsjeyd I have done my best to clarify the page:
* I added some "Notes on Projects and Access"
* I broke up each contributor's access categories using bullet points
* Wherever a group of repositories were referred to, I linked to the corresponding GitHub team, which lists out the exact repos in the group 

I hope that helps!

FYI @sarina (on PTO currently)

Reply:
This seems like a good addition that enough people have commented on already 🙂 

Is there anything specific you'd like me to chime in on, or just an inform?

Reply:
Just an inform!

Reply:
> I have done my best to clarify the page: ...

Thanks a lot @kdmccormick, that's a great improvement! 🚀 

> I wonder if we can avoid any ambiguity by linking you directly to permission listings on GitHub. @itsjeyd , can you see either of these pages, or do you get a 404?
> 
> * https://github.com/orgs/openedx/teams/frontend-all
> * https://github.com/orgs/openedx/people/brian-smith-tcril

Same as @mphilbrick211, frontend-all works fine but brian-smith-tcril 404s for me.